docs/list-of-containers.md

```diff
@@ -0,0 +1,58 @@
+# Images
+
+There's 4 images that you can find in `/images` directory:
+
+- `bench`. It is used for development. [Learn more how to start development](../development/README.md).
+- `production`.
+  - Multi-purpose Python backend. Runs [Werkzeug server](https://werkzeug.palletsprojects.com/en/2.0.x/) with [gunicorn](https://gunicorn.org), queues (via `bench worker`), or schedule (via `bench schedule`).
```

Comment:
Can Werkzeug serve static files too then? We could get rid of Nginx from the image entirely. Then it is up to the image consumer which fronting proxy to us.

https://werkzeug.palletsprojects.com/en/2.2.x/middleware/shared_data/

Reply:
> A WSGI middleware which provides static content for development environments or simple server setups.

it's recommended for simple development

you can use it if you wish, you can mount any py scripts and run. 

I'll keep it as close to upstream as possible for easy maintenance.

Reply:
Can be done even now by whoever knows it. https://stackoverflow.com/a/47628668

things are already complicated for others.

Installing nginx will be challenge if removed from image. I'll prefer keeping it here.